avd-containers/base/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,27 @@
+{
+    "build": {
+        "dockerfile": "./Dockerfile",
+        "context": ".",
+        "args": {
+            "FROM_IMAGE": "${localEnv:FROM_IMAGE}",
+            "FROM_VARIANT": "${localEnv:FROM_VARIANT}"
+        }
+    },
+
+    "features": {
+        "ghcr.io/devcontainers/features/common-utils:2":{
+            // do not install zsh to the base to check size diff
```

Comment:
I'm not sure I understand this comment, as below then you install zsh?

Reply:
Outdated. Was part of early tests.